eclair-node/src/main/scala/fr/acinq/eclair/api/Service.scala

```diff
@@ -279,6 +279,9 @@ trait Service extends ExtraDirectives with Logging {
                         } ~
                         path("usablebalances") {
                           complete(eclairApi.usableBalances())
+                        } ~
+                        path("getdepositaddress"){
```

Comment:
This name indicates that a new address will be generated every time, make it obvious that this is a simple pass-through to bitcoind, and doesn't add ambiguity by introducing  a new concept of `deposit` that's nowhere else in the code.

```suggestion
                        path("getnewaddress"){
```